tools/cldr/cldr-to-icu/src/main/java/org/unicode/icu/tool/cldrtoicu/SupplementalData.java

```diff
@@ -123,7 +124,8 @@ public static SupplementalData create(CldrDataSupplier src) {
                             alias,
                             alias.typeKey.valueFrom(v),
                             alias.replacementKey.valueFrom(v)));
-                } else if (PARENT_LOCALE.matches(v.getPath())) {
+                } else if (PARENT_LOCALE.matches(v.getPath()) && !COMPONENT.optionalValueFrom(v).isPresent()) {
+                    // CLDR-16253 added component-specific parents, which we ignore for now.
```

Comment:
We should handle these properly. Please make this a TODO(ICU-22289) and I'll try to take that issue for icu 73